tests/framework/installer/ceph_manifests.go

```diff
@@ -242,6 +242,14 @@ spec:
       databaseSizeMB: "1024"
 `
 	}
+
+	if m.settings.ConnectionsEncrypted {
+		clusterSpec += `
+  csi:
```

Comment:
@subhamkrai This is causing the smoke and helm suites to fail as found [here](https://github.com/rook/rook/pull/13734#pullrequestreview-1872703859). Please revert these `csi` options in the cluster CR since we don't have that in the 1.12 schema.